packages/entrykit/package.json

```
@@ -0,0 +1,98 @@
+{
+  "name": "@latticexyz/entrykit",
+  "version": "0.0.0",
```

```suggestion
  "version": "2.2.14",
```
for consistency with the other MUD packages (this might be updated automatically on the next release)

---

it does get automatically updated on release (snapshots included), so this initial value doesn't really matter, but will update for consistency